basics0.Rmd

```diff
@@ -17,12 +13,7 @@ parent: Workshop Activities
 
 <img src="images/rstudio-22.png" alt="rstudio logo" style="float:right;width:220px;"/>
 
-```{r setup, include=FALSE}
-knitr::opts_chunk$set(echo = TRUE)
-#uncomment the install.packages if you haven't installed and loaded the 'readxl' package
-#install.packages("readxl")
-library(readxl)
-```
```

Comment:
Moved this section to where you use readexl. I think it might confuse people to start with it

Reply:
this is for the person building the Rmd file. It won't be visible for the workshop taker. I'll add a comment about that.

Reply:
Okay I wasn't sure. I'm a little confused by that workflow